### ISSUE TITLE
Improvement of usability: columns in Score Image

### ISSUE THREAD

**Issue by holgerhennig** [CellProfiler/CellProfiler-Analyst#214]
Current state: only cells in test set are listed when running "Score Image". This can confuse users (as Mark and Anne noted in a previous discussion), in the screenshot the total cell count is 5, but there are 15 cells in the image (10 cells in the training set, 5 cells in the test set).

Suggested target state: same set of columns as in "score all" (but without columns for enrichments or course), i.e. for classes A,B the column headings are:
total cell count, class A cell count, class B cell count, training total cell count, training class A cell count, training class B cell count

Screenshot is current state.

![score_image](https://cloud.githubusercontent.com/assets/5438649/17221357/cf71b1d2-54f3-11e6-8f04-34807b716bf1.png)

**Comment by jhung0** [comment]
added with bcc8b7f

**Comment by holgerhennig** [comment]
tested & works great, thx! Suggest to close the issue
